package.json

```diff
@@ -111,10 +111,12 @@
     "workbox-webpack-plugin": "^5.1.2",
     "worklet-loader": "^1.0.0"
   },
-  "homepage": "https://ury.org.uk/webstudio",
+  "homepage": "./",
```

Comment:
This broke the automated Jenkins builds. @mstratford is this what it needs to be for baps3? If so it's fine, we'll just need to amend the jenkins scripts.

Reply:
hmm, will check, but I think it was when making a build, the links to resources would be all wrong.

Reply:
I’ve worked out a way to have it be more reliable - it’s live on my manual Jenkins job, will merge into the Jenkinsfile